packages/lib/gpc/src/gpcChecks.ts

```diff
@@ -455,6 +476,25 @@ export function checkProofInputsForConfig(
         }
       }
 
+      if (entryConfig.notEqualsEntry !== undefined) {
```

Comment:
I think this block is identical to the block above, modulo a few differences which could be variables.  Given the config makes the two options mutually exclusive above, I'd consider combining them to avoid code bloat.  That's a stylistic preference, though, so not a big deal if you disagree.